benchmark.py

```diff
@@ -576,6 +576,7 @@ def run_throughput_test(query_root, data_dir, host, port, db_name, user, passwor
         for i in range(num_streams):
             stream = i + 1
             # refresh functions
+            result = Result("ThroughputRefreshStream")
```

Comment:
This will not work. You are recreating the object within the loop.

Reply:
moved it outside of the loop now.